static/js/filter.js

```diff
@@ -0,0 +1,37 @@
+import { render } from './partials/render.js'
+
+const input = document.querySelectorAll('fieldset input')
+
+function getFilterdata () {
+  return new Promise(async (resolve, reject) => {
+    const lastValue = window.location.pathname.split('/').slice(-1)[0].split('-').slice(-1)[0]
+    const url = window.location.protocol + '//' + window.location.host + '/' + 'filter' + '/' + lastValue
+
+    try {
+      const res = await fetch(url)
+      const data = await res.json()
+      resolve(data)
+    } catch (err) {
+      reject(err)
+    }
+  })
+}
+
+async function dataFilter (value) {
```

Comment:
I.m.o. this function is unnecessary as it only passes through a functions result

Reply:
Ye i think it used to do something else, but i have removed it